python/Create_Large_Test_Files.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Creating Large HMDA Submission Test Files
+# 
+# This notebook demonstrates a class for creating large HMDA submission test files.
+
+# In[ ]:
+
+
+#The following code imports the required packages.
+import math
+import json
+import os
+import pandas as pd
+import random
+import string
+import time
+import yaml
+import shutil as sh
+
+from lar_generator import lar_gen #Imports lar_gen class. 
+lg = lar_gen() #Instantiates lar_gen class as lg. 
+
+
+# ### Creating Clean Test Files
+# 
+# The following defines a function that creates clean test files. It uses an existing clean file to create a new clean file of any row count. This set of functions is from the Clean_Files class defined below. An existing clean test file can be separated into its Transmittal Sheet and LAR components, using the Clean_Files class function separate_lar_ts. 
+
+# In[ ]:
+
+
+cf = Clean_Files()
+cf.separate_lar_ts(file = "../edits_files/file_parts/clean_file_1000_rows_Bank1.txt")
+
+
+# ### Creating Clean Test Files With Specified Counts in a List
+# A loop may be used to create clean large test files with the counts in a list. 
+
+# In[ ]:
+
+
+rows = [297]
+
+for row in rows:
+    ts_file = "../edits_files/file_parts/ts_data.txt" #Stores the file path for TS data as ts_file. 
+    lar_file = "../edits_files/file_parts/lar_data.txt" #Stores the file path for LAR data as lar_file.
+    df = cf.get_rows(lar_file = lar_file, row_count = row)
+    df = cf.assign_uli(df=df, ts_file = ts_file)
+    cf.lar_to_hmda_file(ts_file=ts_file, lar_rows_df=df)
+
+
+# In[ ]:
+
+
+class Clean_Files(object):
```

Comment:
Let's move this class to it's own .py file and just import it to this script. That will make it follow the design of the existing code base.